### DISCUSSION TITLE
AI generated commit/release info

### DISCUSSION THREAD

**Discussion by chrisbbreuer** [Zhengqbbb/cz-git#91] [Ideas]
Hi everyone!

With tools & ideas like [aicommits](https://npmjs.com/package/aicommits) and [wtfcommit](https://twitter.com/marcelpociot/status/1573060174520389632?lang=en), I think it would make sense to allow for even further automation. I could see scenarios where `type`, `scope`, and `messages` can be correctly interpreted by AI, and even release diffs 🙂

What do you all think? Maybe via hooks we could extend it?

**Comment by Zhengqbbb** [comment]
Yep! ! ! I also noticed this project today, I looked at the code, I think I can add this feature and do better, but today I think more about how to do interact and configure enhancements,
Anyway this is my next plan!


<img width="731" alt="image" src="https://user-images.githubusercontent.com/40693636/218806823-46a2732c-1776-49c7-9d7a-3ce8d4b6c091.png">

**Reply by chrisbbreuer** [reply]
Yay! This sounds like such a cool feature. I happily volunteer to test anything haha

Looking very much forward to using it as part of [Stacks](https://github.com/stacksjs/stacks) because otherwise, my commit log seemingly will not change in a while 😂  

<img width="423" alt="image" src="https://user-images.githubusercontent.com/6228425/218807761-d9e61d1d-5c98-44c1-873f-67ae72462dac.png">

**Comment by Zhengqbbb** [comment]
I wish it was the weekend now. I already feel ready to do it.🧐🧐🧐


But now I'm in China, so I have to find a way to register an OpenAI account tomorrow. And, I also began to prepare a BING account appointment.

**Comment by Zhengqbbb** [comment]
Let move and follow this issue to next discussion: #92

**Comment by Zhengqbbb** [comment]
See Demo: 
https://twitter.com/zhengqbbb/status/1625765540458557440

**Comment by Zhengqbbb** [comment]
@chrisbbreuer 
The latest version 🌟(v1.5.0)🌟 is supported OpenAI auto generate subject.
See guide page: https://cz-git.qbb.sh/recipes/openai
Please have a try. 👀 Let me know if there are still any issues. Thanks for you support! 🎉
